### ISSUE TITLE
[To confirm] http11-for-imagify not compatible with PHP 7.4

### ISSUE THREAD

**Issue by jbma** [wp-media/imagify-plugin#476]
Haven't been able to confirm it, but it looks like that the custom plugin we provide to solve the chunky upload issue isn't compatible with PHP 7.4

[Source](https://secure.helpscout.net/conversation/1048618529/139627/)

**Comment by jonathandhn** [comment]
It woks on my side, on both Debian Buster PHP7.4.1 and LiteSpeed PHP7.4.1

**Comment by Screenfeed** [comment]
@jbma So you get the "chunky" error with HTTP/1.0 and HTTP/1.1?

**Comment by Screenfeed** [comment]
Related #465

**Comment by jonathandhn** [comment]
Of course, I use this one from [here](https://github.com/wp-media/imagify-plugin/issues/465) . I moved from a web server to another, still the same. No plugging set on, no uploads to Imagify.

**Comment by Screenfeed** [comment]
@jonathandhn So it doesn't work for you neither? You said in your previous message that it was working.

**Comment by jonathandhn** [comment]
Yes, if plugin "helper" is set on, it's working. 
(deb10u2, curl 7.64.0, OpenLiteSpeed PHP 7.4.1, OpenLiteSpeed 1.6.5)

**Comment by Screenfeed** [comment]
OK thank you.
